examples/showcase-all-resources.tf

```diff
@@ -0,0 +1,105 @@
+# App
+
+resource "ably-app" "app1" {
+  status                 = "enabled"
```

Comment:
We're missing the `name` parameter in this one.

Reply:
Will add shortly.

Reply:
Done 7b4831880da8027c63f67cb01aa64f9a029b0699